server/src/rooms/BattleRoom.ts

```diff
@@ -84,8 +96,8 @@ export class BattleRoom extends Room<MyRoomState> {
     const player = new Player();
 
     // place Player at a random position
```

Comment:
the comment seems a bit misleading, i suppose its fixed position?

Reply:
my bad forgot to remove this comment.